WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/ProductInputTransformer.swift

```diff
@@ -41,6 +41,58 @@ struct ProductInputTransformer {
         return order.copy(items: items)
     }
 
+    /// Adds, deletes, or updates order items based on the multiple given product inputs
+    ///
+    /// - Parameters:
+    ///   - input: Array of types of products the synchronizer supports
+    ///   - order: Represents an Order entity.
+    ///   - updateZeroQuantities: When true, items with `.zero` quantities will be updated instead of being deleted.
+    ///
+    /// - Returns: An Order entity.
+    static func updateMultiple(input: [OrderSyncProductInput], on order: Order, updateZeroQuantities: Bool) -> Order {
+        var items = order.items
+
+        for productInput in input {
+            items.append(contentsOf: updateOrderItems(
+                from: productInput,
+                order: order,
+                updateZeroQuantities: updateZeroQuantities)
+            )
+        }
+
+        return order.copy(items: items)
+    }
+
+    /// Updates the `OrderItems` array with `OrderSyncProductInput`.
+    /// Uses the same implementation as `update()` but returns an array of OrderItems instead,
+    /// rather than aggregating them into the Order
+    ///
+    /// - Parameters:
+    ///   - input: Types of products the synchronizer supports
+    ///   - order: Represents an Order entity.
+    ///   - updateZeroQuantities: When true, items with `.zero` quantities will be updated instead of being deleted.
+    /// - Returns: An array of Order Item entities
+    private static func updateOrderItems(from input: OrderSyncProductInput,
```

Comment:
Super not a blocker, we can fix the indentation of the parameters here to make all of them aligned to the first one

Reply:
The indentation for these parameters comes automatically from Xcode's Code Actions > Add documentation, the same indentation is applied in other places of the codebase where we use the same 😅 

I did add extra spacing for the `- Returns` documentation thought! Here: https://github.com/woocommerce/woocommerce-ios/pull/9173/commits/8621a80703f284e585e654ad930451b514b5015a

Reply:
No, i meant for the function name:

```
private static func updateOrderItems(from input: OrderSyncProductInput,
                                         order: Order, updateZeroQuantities: Bool) -> [OrderItem] {
```

`order` and `updateZeroQuantities` are not aligned to the first parameter

Reply:
Oh! I see! I fixed the alignment here: https://github.com/woocommerce/woocommerce-ios/pull/9173/commits/16fcc9380e06a1dd8f060e1aec037749e35bd7af